libs/blocks/marketo/marketo.js

```diff
@@ -97,37 +97,39 @@ const init = (el) => {
   const baseURL = formData[BASE_URL];
   const munchkinID = formData[MUNCHKIN_ID];
 
-  if (!formID || !baseURL) {
+  setPreferences(formData);
+
+  if (!formID || !baseURL || !munchkinID) {
     el.style.display = 'none';
     return;
   }
 
-  loadScript(`https:${baseURL}/js/forms2/js/forms2.min.js`)
-    .then(() => {
-      const { MktoForms2 } = window;
-      if (!MktoForms2) throw new Error('Marketo forms not loaded');
+  const fragment = new DocumentFragment();
+  const formWrapper = createTag('section', { class: 'marketo-form-wrapper' });
 
-      const fragment = new DocumentFragment();
-      const formWrapper = createTag('section', { class: 'marketo-form-wrapper' });
+  if (formData.title) {
+    const title = createTag('h3', { class: 'marketo-title' }, formData.title);
+    formWrapper.append(title);
+  }
```

Comment:
there tends to be newlines after if blocks

Reply:
Thanks